okhttp-clients/src/main/java/com/palantir/conjure/java/okhttp/QosExceptionResponseHandler.java

```diff
@@ -29,7 +29,7 @@
 import org.slf4j.LoggerFactory;
 
 /**
- * A {@link ResponseHandler} that turns HTTP responses pertaining to http-remoting {@link QosException}s into {@link
+ * A {@link ResponseHandler} that turns HTTP responses pertaining to conjure-java-api {@link QosException}s into {@link
```

Comment:
i think this is redundant since a link is already provided.

Reply:
rephrased this as it just linked to the same thing twice, making it confusing to read.